pkg/handler/handler_test.go

```diff
@@ -138,17 +130,16 @@ func TestLatestReadme(t *testing.T) {
 			Title: "Two",
 		},
 	}
-	r := resolver{readmes: rs}
-	first, err := r.LatestReadme()
+
+	first, err := LatestReadme(rs)
```

Comment:
So this should no longer test the real resolvers?

Reply:
The resolver is dependent on the google storage libs. Ref the comment above, I want to refactor the code so that this is not the case and we can run tests with test data, but as you said, I thought we should get this in if it's time-sensitive.